static/src/javascripts/projects/common/modules/email/email.js

```diff
@@ -287,7 +287,10 @@ const submitForm = (
                     method: 'post',
                     body: data,
                     headers: {
+                        Credentials: 'include',
                         Accept: 'application/json',
+                        'Content-Type':
+                            'application/x-www-form-urlencoded; charset=utf-8',
```

Comment:
is this strictly necessary?

Reply:
it's apparently best practice (and is also set by the other working CSRF check in edit profile, though that sets it automatically using the `FormData` class).

I also think it 'hints' to the play csrf checker to check the form body.